Detectors/ITSMFT/MFT/tracking/include/MFTTracking/TrackerConfig.h

```diff
@@ -46,27 +55,41 @@ class TrackerConfig
   Float_t mLTFclsR2Cut;
   Float_t mROADclsRCut;
   Float_t mROADclsR2Cut;
-  Int_t mLTFseed2BinWin;
-  Int_t mLTFinterBinWin;
   Int_t mRBins;
   Int_t mPhiBins;
   Int_t mRPhiBins;
-  Float_t mRBinSize;
-  Float_t mPhiBinSize;
-  Float_t mInverseRBinSize;
-  Float_t mInversePhiBinSize;
+  Float_t mZVtxMin;
+  Float_t mZVtxMax;
+  Float_t mRCutAtZmin;
   Bool_t mLTFConeRadius;
   Bool_t mCAConeRadius;
-
   /// Special track finder for TED shots and cosmics, with full scan of the clusters
   bool mFullClusterScan = false;
+  Float_t mTrueTrackMCThreshold; // Minimum fraction of correct MC labels to tag True tracks
 
-  ClassDefNV(TrackerConfig, 2);
+  static Float_t mPhiBinSize;
```

Comment:
@rpezzi what is the reason to have still relatively large arrays as static? In this case they are eating up memory in the every process the MFT tracking is linked to (and at the moment a lot of devices). Cannot you init them during the tracking initializing?

Reply:
Hi @shahor02 , these arrays are static so that every tracker thread can use the same, but indeed these could be initialized at runtime by the tracker. Unfortunately I am away and won't be able to work on that.

Another point, `mBins` and `mBinsS` are in fact identical with the current tracker. Therefore one can be suppressed.